Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.11
+
+RUN apk add curl
+
+RUN apk add python3
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+
+COPY . .
+
+RUN pip3 install flask
+
+RUN source $HOME/.poetry/env && poetry update && poetry install
+
+RUN pip3 install gunicorn
+
+RUN pip3 install requests
```

Comment:
We shouldn't need pip at all as we're using poetry to manage our dependencies, these should be included as dependencies, and we can then run the app with e.g. `poetry run gunicorn ...`

Reply:
It didn't like running gunicorn otherwise and when I checked online this is what they wanted

Reply:
You shouldn't be using pip. You are using poetry to manage dependencies (and their versions).

Run `poetry add gunicorn` one time in your terminal to update pyproject.toml as necessary. And then get rid of all of these `pip3 install` commands

You will need `poetry run gunicorn ...` just like you need `poetry run flask run ...`
Alternatively, and slightly better:
Run `poetry config virtualenvs.create false --local` just before "poetry install" so that it will install packages globally instead of using a virtual environment